lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -482,6 +486,32 @@ data MkFragment
         ChimericAccount
         (XPrv, Passphrase "encryption")
 
+getPoolRetirement
+    :: Get (PoolId, Tx)
+getPoolRetirement = error "retirement not implemented"
+
+getPoolRegistration
+    :: Get (PoolId, [PoolOwner], Tx)
+getStakeDelegation = do
+    label "getPoolRegistation" $ do
+        _serial <- getByteString 16
+        _perms <- getByteString 8
+        _keys <- getByteString 256 -- fixme: check POOL-KEYS = VRF-PUBLICKEY KES-PUBLICKEY
```

Comment:
**[suggestion]**

Use [skip](https://hackage.haskell.org/package/binary-0.10.0.0/docs/Data-Binary-Get.html#v:skip) with a clear comment about what's being skipped.

**Why**
Most of these doesn't really deserialize to `ByteString` and I suppose, this is really what you intended to write ^^"